test/clj/rems/browser_test_util.clj

```diff
@@ -171,6 +170,21 @@
 
 ;;; etaoin extensions
 
+;; TODO our input fields process every character through re-frame.
+;; Etaoin's fill-human almost works, but very rarely loses characters,
+;; probably due to the lack of a _minimum_ delay between keypresses.
+;; This is a reimplementation.
+(def +character-delay+ 0.1)
+
+(defn fill-human [q text]
+  (doseq [c text]
+    (et/wait +character-delay+)
+    (et/fill (get-driver) q c))
+  (et/wait +character-delay+)
+  (let [value (get-element-attr q "value")]
+    (when-not (= text value)
+      (log/warn "Failed to fill field to" (pr-str text) "got" (pr-str value) "instead."))))
```

Comment:
I would like to see the typos and random delays here too. Is there a reason why you did not just copy them too?

Reply:
etaoin is under EPL, which is a copyleft license, so I didn't want to copy any code from it

Reply:
I can reimplement typos if you want :)

Reply:
I think it would be nice.